desktop/apps/auction2/components/page/Banner.js

```diff
@@ -0,0 +1,75 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import get from 'lodash.get'
+
+export default function Banner (props) {
+  const {
+    auction,
+    isLiveOpen,
+    liveAuctionUrl
+  } = props
+
+  const {
+    cover_image,
+    name
+  } = auction.toJSON()
+
+  const BackgroundBanner = ({ children }) => {
+    const imageUrl = get(cover_image, 'cropped.url', '')
+
+    return (
+      <div
+        className='auction-banner-live-bg'
+        style={{ backgroundImage: `url('${imageUrl}')` }}
+        alt={name}
+      >
+        {children}
+      </div>
+    )
+  }
+
+  BackgroundBanner.propTypes = {
+    children: PropTypes.node
+  }
+
+  return (
+    <div>
+      {isLiveOpen
+        ? <div className='auction-banner auction-banner-live-open'>
+          <BackgroundBanner />
+
+          <div className='auction-banner-live-details'>
+            <h1>
+              Live Bidding Now Open
+            </h1>
+
+            <a href={liveAuctionUrl} className='avant-garde-button-white'>
+              Enter live auction
+            </a>
+          </div>
+        </div>
+        : <div className='auction-banner'>
+          <BackgroundBanner>
+            <div className='auction-clock white-overlay-clock js-auction-clock'>
+
+              {/*
+                Backbone-based view:
+                components/clock/index.coffee
```

Comment:
Food for thought—could we instead combine the lifecycle method approach above with some minimal rendering like this into it's own react component inside the clock components instead?

desktop/components/clock/react.js
```javascript
class Clock extends Component {
  componentDidMount() {
    this.view = new Clock(...)
  }
  componentWillUnmount() {
    this.view.remove()
  }
  render() {
    <div className='clock>...</div>
  }
}
```
import Clock from 'desktop/components/clock/react.js'

<Clock></Clock>
```

That way we can colocate the react integration and make it easier to refactor these view towards react within the scope of the component instead of refactoring across apps. I could easily be overlooking something though.